src/managers/RoleManager.js

```diff
@@ -87,7 +87,10 @@ class RoleManager extends BaseManager {
    * Creates a new role in the guild with given information.
    * <warn>The position will silently reset to 1 if an invalid one is provided, or none.</warn>
    * @param {Object} [options] Options
```

Comment:
Can't this extend RoleData?

Reply:
This is my first PR to the library, as such I am following the pattern of the other managers where they extend BaseManager.

Reply:
Maybe I misunderstood this...Which part could extend RoleData? The RoleManager class, or another part?

Reply:
The Options object itself, so in the docs, it's just an extension of RoleData